src/components/Certify/Certify.vue

```diff
@@ -238,8 +238,13 @@ export default class Certify extends Vue {
   font-weight: normal;
 }
 
+// Vuetify overrides
 ::v-deep .v-label {
   color: $gray7;
   font-weight: normal;
 }
+
+::v-deep .v-input--is-disabled input, .v-input--is-disabled textarea {
```

Comment:
How about putting the second selector on a new line, so it stands out?

Reply:
For sure! Referring to the `.v-input--is-disabled textarea`?

Reply:
Updated